exercises/nucleotide-count/description.md

```diff
@@ -1,11 +1,19 @@
-Given a single stranded DNA string, compute how many times each nucleotide occurs in the string.
+Most people inherit not only their socioeconomic class from their parents, but also DNA (deoxyribonucleic acid), a set of chemical instructions that influence how their bodies are constructed!
+DNA is a long chain of other chemicals and the most important are the four nucleotides, Adenine, Cytosine, Guanine and Thymine. A single DNA chemical can contain billions of these four nucleotides and the order in which they occur is important!
+We call the order of these nucleotides in a bit of DNA a "DNA sequence".
 
-The genetic language of every living thing on the planet is DNA.
-DNA is a large molecule that is built from an extremely long sequence of individual elements called nucleotides.
-4 types exist in DNA and these differ only slightly and can be represented as the following symbols: 'A' for adenine, 'C' for cytosine, 'G' for guanine, and 'T' thymine.
+We represent a DNA sequence as an ordered collection of these four nucleotides and a common way to do that is with a string of characters such as "ATTACA" for a DNA sequence of 6 nucleotides.
 
-Here is an analogy:
-- twigs are to birds nests as
-- nucleotides are to DNA as
-- legos are to lego houses as
-- words are to sentences as...
+Given a string representing a DNA sequence, count how many of each nucleotide is present.
+If the string contains characters that aren't A, C, G, or T then it is invalid and you should throw an error.
+
+For example:
+
+```
+"GATTACA" -> 'G': 1, 'A': 3, 'T': 2, 'C': 1
+"INVALID" -> error
+```
```

Comment:
I'd argue that "you should throw an error" is a term that does not apply to all tracks. Some tracks will return an error code or even an error type.  My suggestion would be to either rephrase a bit.

Reply:
Yeah. But everything before it is good.

Is invalid will hint at "special case".

Reply:
Changed to read "signal an error"